docs/user-guide.md

```diff
@@ -752,6 +753,21 @@ The Spark Operator provides limited support for resource quota enforcement using
 
 If you are running Spark applications in namespaces that are subject to resource quota constraints, consider enabling this feature to avoid driver resource starvation. Quota enforcement can be enabled with the command line arguments `-enable-resource-quota-enforcement=true`. It is recommended to also set `-webhook-fail-on-error=true`.
 
+## Running multiple instances of the operator within the same k8s cluster
```

Comment:
Please use the capital case for the first letter of the words.

Reply:
Done.